services/user/CommonApi/src/CommonApi.cpp

```diff
@@ -79,6 +105,29 @@ namespace SystemService
                 .body        = psio::convert_to_frac(trx),
             };
          }
+         if (request.target == "/common/set-auth-cookie")
+         {
+            request.body.push_back(0);
+            psio::json_token_stream jstream{request.body.data()};
+            auto                    params = psio::from_json<cookie_data>(jstream);
+
+            std::vector<HttpHeader> headers;
+            bool                    isDevChain = psibase::isDevChain(request);
+            std::string             cookieName = "__Host-SESSION";
+            
+            std::string cookieAttribs =
+                "Path=/; HttpOnly; Secure; SameSite=Strict; Expires=" + getCookieExpiration();
```

Comment:
`Max-Age` is simpler than `Expires`, because it can just be a literal. It also doesn't assume that the client and server have synchronized clocks.